com.github.nrittsti.NTag.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.github.nrittsti.NTag
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: ntag
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
+modules:
+  - name: ntag
+    buildsystem: simple
+    build-commands:      
+      - install -dm755 /app/opt/ntag/lib
+      - install -Dm644 com.github.nrittsti.NTag.desktop /app/share/applications/com.github.nrittsti.NTag.desktop
+      - install -Dm644 com.github.nrittsti.NTag.metainfo.xml /app/share/metainfo/com.github.nrittsti.NTag.metainfo.xml
+      - install -Dm644 ntag.png /app/share/icons/hicolor/128x128/apps/com.github.nrittsti.NTag.png
+      - install -Dm755 com.github.nrittsti.NTag.sh /app/bin/ntag      
+      - install -Dm644 lib/*.jar /app/opt/ntag/lib
+      - install -Dm644 *.{jar,png,properties} /app/opt/ntag      
+      - install -dm755 /app/opt/ntag/jre
+      - cp -r --no-preserve=ownership --preserve=mode jre/* /app/opt/ntag/jre/
+    sources:
+      - type: archive
+        url: https://github.com/nrittsti/ntag/releases/download/v1.2.8/ntag-1.2.8-linux_bin.tar.gz
+        sha256: 4a1c32c333f471b944f7da9ce08221e92f8ac873056abb562b16f88cd7159d3f
+      - type: file
+        path: com.github.nrittsti.NTag.desktop        
+      - type: file
+        path: com.github.nrittsti.NTag.sh
+      - type: file
+        path: com.github.nrittsti.NTag.metainfo.xml        
```

Comment:
Ideally these three files should be included into your GitHub repo and tarball as they are not specific to Flathub, but I'll the choice to you.

Reply:
i have used other flathub repos as a template, but i agree with you